torch/_inductor/cudagraph_utils.py

```diff
@@ -178,3 +179,17 @@ def get_placeholder_stack_trace(placeholder: torch.fx.Node) -> Optional[str]:
             return user.stack_trace
 
     return None
+
+
+class CheckInvariantStatus(Enum):
+    # Check invariant succeeded
+    SUCCESS = 1
+
+    # Previously managed data pointers are not stable
+    CudagraphManagedIdxMismatch = 2
```

Comment:
nice! someone has actually run into this previously and it was inscrutable to them. could we add this to the pr ?